src/sql/src/connectors.rs

```diff
@@ -0,0 +1,218 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use mz_repr::GlobalId;
+use mz_sql_parser::ast::{
+    AstInfo, AvroSchema, CreateSourceConnector, CreateSourceFormat, CreateSourceStatement,
+    CsrConnector, CsrConnectorAvro, CsrConnectorProto, Format, KafkaConnector,
+    KafkaSourceConnector, ProtobufSchema, UnresolvedObjectName,
+};
+
+use crate::normalize::unresolve;
```

Comment:
I had thought `cargo fmt` handled getting the imports right since I let automation do all of the adding and arranging of those. I also thought I had de-qualified everything that was imported but apparently I missed some once it stopped triggering the unused import rules. Thanks for catching this!